wallet/walletrpc.c

```diff
@@ -506,11 +506,13 @@ static struct command_result *json_listfunds(struct command *cmd,
 							  "short_channel_id",
 							  c->scid);
 
-			json_add_amount_sat(response,
-					    amount_msat_to_sat_round_down(c->our_msat),
-					    "channel_sat", "our_amount_msat");
-			json_add_amount_sat(response, c->funding,
-					    "channel_total_sat", "amount_msat");
+			json_add_amount_sat_compat(response,
+						   amount_msat_to_sat_round_down(c->our_msat),
+						   "channel_sat",
+						   "our_amount_msat");
+			json_add_amount_sat_compat(response, c->funding,
+						   "channel_total_sat",
+						   "amount_msat");
```

Comment:
... same here in line 512 and 515, since this msat/sat is mixed up everywhere, maybe its just me??